example/src/components/SprakVelger.tsx

```diff
@@ -3,7 +3,7 @@ import React from "react";
 interface Props {
   locales: string[];
   valgtLocale: string;
-  settValgtLocale: (locale: "en" | "nb") => void;
```

Comment:
Definere en enum type "Locales", så vi slipper å gjenta dette så mange ganger (med muligheter for feil)?

Reply:
Fixed!